scuba/scuba.py

```diff
@@ -154,6 +154,26 @@ def add_volume(self, hostpath, contpath, options=None):
             options = []
         self.volumes.append((hostpath, contpath, options))
 
+    def try_create_volumes(self):
+        '''Try to create non-existent host paths prior to docker run invocation
```

Comment:
It might be worthwhile to note that this only creates *user-configured* volumes (either via command-line option of via `.scuba.yml`) and not any scuba-internal volumes/mounts.